Delver/Assets/Source/Gameplay/Abilities/Ability_SimpleMelee.cs

```diff
@@ -33,11 +32,7 @@ public override void UpdateAbility(Actor abilityUser)
 
         if(currentCollider != null)
         {
-            if(!stopMovement && (currentCollider.transform.position - activatedPosition).sqrMagnitude < (maxRange * maxRange))
-            {
-                currentCollider.transform.position += activatedDirection * colliderSpeed * Time.deltaTime;
-            }
-            else
+            if (stopMovement && (currentCollider.transform.position - activatedPosition).sqrMagnitude < (maxRange * maxRange))
```

Comment:
I initially had this as ||, but when you first fire the collider it immediately satisfies the second condition because the currentPositon and the activatedPosition are the same (so current - activated = 0).